apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -263,7 +269,11 @@ export class TeacherFeedback extends Component {
                   <RubricField
                     key={level}
                     showFeedbackInputAreas={showFeedbackInputAreas}
-                    expandByDefault={this.props.displayKeyConcept}
+                    expandByDefault={
```

Comment:
this is calculation is getting pretty complex, so let's assign to a variable at the top of the `render()` method. something like:
```
const expandByDefault = this.props.displayKeyConcept || (expandPerformanceLevelForStudent && this.state.performance === level)
```

Reply:
similar to what you're doing for `expandPerformanceLevelForStudent`

Reply:
The level here comes from the map which is on the line right above <RubricField. In order to move this out I think I would need to make it a function. Should I do that?

Reply:
The level here comes from the map which is on the line right above <RubricField. In order to move this out I think I would need to make it a function. Should I do that?

Reply:
ah, sorry, i should've taken a closer look at the context here. in that case, it's fine to leave as-is